guide/additional-info/miscellaneous-examples.md

```diff
@@ -0,0 +1,161 @@
+
+```
+npm install --save ytdl-core
+```
+
+```js
+const Discord = require('discord.js');
+const ytdl = require('ytdl-core');
+
+const client = new Discord.Client();
+
+client.on('message', message => {
+	if (message.content.startsWith('!play')) {
+		if (message.channel.type !== 'text') return;
+
+		const { voiceChannel } = message.member;
+
+		if (!voiceChannel) {
+			return message.reply('please join a voice channel first!');
+		}
+
+		voiceChannel.join().then(connection => {
+			const stream = ytdl('https://www.youtube.com/watch?v=D57Y1PruTlw', { filter: 'audioonly' });
+			const dispatcher = connection.playStream(stream);
+
+			dispatcher.on('end', () => voiceChannel.leave());
+		});
+	}
+});
+
+client.login('your-token-goes-here');
+```
+
+### Catching UnhandledPromiseRejectionWarnings
+
+If you've ever seen something labled `UnhandledPromiseRejectionWarnings` in your console, it can sometimes be a bit vague as to why it occured. In addition to that, not handling promise rejections is currently deprecated and will eventually be removed. Once that happens, any unhandled promise rejections will close your app with a non-zero exit code.
+
+You can use a single line of code to prevent that, though. This will also display the line number of where the error occured.
+
+```js
+process.on('unhandledRejection', error => console.error(`Uncaught Promise Error:\n${error.stack}`));
+```
+
+### Sending an embed
+
+#### RichEmbed builder
+
+```js
+// somewhere at the top of your file
+const Discord = require('discord.js');
+
+// somewhere inside a command, an event, etc.
+const exampleEmbed = new Discord.RichEmbed()
+	.setColor('#0099ff')
+	.setTitle('Some title')
+	.setURL('https://discord.js.org/')
+	.setAuthor('Some name', 'https://i.imgur.com/wSTFkRM.png', 'https://discord.js.org')
+	.setDescription('Some description here')
+	.setThumbnail('https://i.imgur.com/wSTFkRM.png')
+	.addField('Regular field title', 'Some value here')
+	.addBlankField()
+	.addField('Inline field title', 'Some value here', true)
+	.addField('Inline field title', 'Some value here', true)
+	.addField('Inline field title', 'Some value here', true)
+	.setImage('https://i.imgur.com/wSTFkRM.png')
+	.setTimestamp()
+	.setFooter('Some footer text here', 'https://i.imgur.com/wSTFkRM.png');
+
+message.channel.send({ embed: exampleEmbed });
+```
+
+##### RichEmbed builder notes
+
+* If you're on discord.js v11.2 or above, you can simply do `message.channel.send(exampleEmbed)`.
+* For `.setColor()`, it accepts a base 10 integer, HEX color as a string, or an array of RGB values.
+* There are also specific strings you can use with `.setColor()`, e.g. `DARK_RED` or `RANDOM`. A list of them can be found [here](https://discord.js.org/#/docs/main/stable/typedef/ColorResolvable).
+* `.addBlankField()` is just shorthand for `.addField('\u200b', '\u200b')`. If you want to make it inline as well, pass in `true` as the first argument.
```

Comment:
You could possibly note that the formatting might be not ideal. Perhaps suggest the invisible thumbnail workaround. Or not if you want. Just a thought.

Reply:
Something that just now came to mind: since this section contains a bit too much info to be considered a "quick" example, once the embeds page is created, we can strip the info from this page and move it there, and then leave a message saying "Hey, check out this page for more info on embeds" or something along those lines.